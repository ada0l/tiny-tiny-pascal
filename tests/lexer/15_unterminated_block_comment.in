a := a + 3 + 4;
{ Hello, world
a := a + 3 + 4;
