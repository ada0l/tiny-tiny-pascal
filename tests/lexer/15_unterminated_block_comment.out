1	1	Id	a	a
1	3	Operation	ASSIGN	:=
1	6	Id	a	a
1	8	Operation	ADD	+
1	10	Integer	3	3
1	12	Operation	ADD	+
1	14	Integer	4	4
1	15	Separator	SEMICOLON	;
Exception: Line: 2; Column: 1; Unterminated block comment